frontend/src/views/files/Listing.vue

```diff
@@ -882,9 +887,9 @@ export default {
       const windowHeight = window.innerHeight;
 
       // Quantity of items needed to fill 2x of the window height
-      const showQuantity = Math.ceil(
-        (windowHeight + windowHeight * 2) / this.itemWeight
-      );
+      const showQuantity = window.sessionStorage.getItem(this.$route.path) ?
+        Math.ceil((JSON.parse(window.sessionStorage.getItem(this.$route.path))[0] + windowHeight * 2) / this.itemWeight) :
+        Math.ceil((windowHeight + windowHeight * 2) / this.itemWeight);
```

Comment:
This is important. Without it, the scrollTop fucntion can reach to the desired position in Chrome, with a obvious scrolling process. and in firefox and safari, the scrollTop function can't reach to the desired position because it runs very fast in firefox and can only invoke the scrollevent() once or twice.  To solve this, some coders may save this page's whole vue instance to window.sessionstorage, it's too complicated.  

With this modification, the scrolltop works perfectly. it can reach to the desired position immidiately without invoking the scrollEvent().